include/pocl_context.h

```diff
@@ -31,25 +31,25 @@
 #include "pocl_types.h"
 
 struct pocl_context32 {
-  uint work_dim;
```

Comment:
Why?

Reply:
This was a micro-optimization done as part of larger work (which was already merged by you earlier, somewhere in the specialization work IIRC). So essentially a "leftover".